app/admin/allocations.rb

```diff
@@ -89,7 +89,7 @@
                                       .select(:id, :name)
 
         input :user, as: :select, collection: company_users_not_allocated
-        input :project, collection: current_user.company.projects.active.order(:name)
+        input :project, collection: (current_user.company.projects.active.to_a | [@resource.project]).reject(&:blank?).sort_by(&:name)
```

Comment:
can you add tests?

Reply:
sure

Reply:
should i create a allocations_spec.rb in spec/admin?
Or there is another place designed to test this issue?